ui/components/multichain/account-list-item/account-list-item.js

```diff
@@ -69,7 +70,7 @@ export const AccountListItem = ({
   connectedAvatar,
   connectedAvatarName,
   isPinned = false,
-  showOptions = false,
+  showOptionTypes = MenuOptionTypes.None,
```

Comment:
`showOptionTypes` doesn't seem like an intuitive prop name.  Can we use something like `menuType`?

Reply:
Yup https://github.com/MetaMask/metamask-extension/pull/23232/commits/f04561a22fb16e2948e6b45218a8857144784531